lightning/src/ln/channel.rs

```diff
@@ -2604,18 +2606,9 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 						return Err(ChannelError::Close("Peer sent a garbage channel_reestablish with secret key not matching the commitment height provided"));
 					}
 					if msg.next_remote_commitment_number > INITIAL_COMMITMENT_NUMBER - self.cur_local_commitment_transaction_number {
-						self.latest_monitor_update_id += 1;
-						let monitor_update = ChannelMonitorUpdate {
-							update_id: self.latest_monitor_update_id,
-							updates: vec![ChannelMonitorUpdateStep::RescueRemoteCommitmentTXInfo {
-								their_current_per_commitment_point: data_loss.my_current_per_commitment_point
-							}]
-						};
-						self.channel_monitor.as_mut().unwrap().update_monitor_ooo(monitor_update.clone()).unwrap();
-						return Err(ChannelError::CloseDelayBroadcast {
-							msg: "We have fallen behind - we have received proof that if we broadcast remote is going to claim our funds - we can't do any automated broadcasting",
-							update: monitor_update
-						});
```

Comment:
Doesn't matter anymore now but did we forget to implement "my_current_per_commitment_point" does not match the expected value"? I'm not even sure you can verify this given you don't have a comparison base if you're fallen behind.

Reply:
Hmm, yea, seems so. One of those awkward "checking this has no value except to enforce that people don't generally set it wrong".........